chapters/synchronous.tex

```diff
@@ -1118,44 +1118,45 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 However, even if relations do not generate events, when-clauses and operators \lstinline!edge! and \lstinline!change! should behave as normal.
 \end{nonnormative}
 
-From the view of the clocked partition, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
-Therefore, such a partition is handled in the same way as any other clocked partition.
-Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized continuous-time partition with other partitions.
-Hereby, a discretized continuous-time partition is seen as a clocked partition.
+From the view of other partitions, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
+Therefore, such a partition is externally treated as any other clocked partition.
+Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized partition with other partitions.
```

Comment:
Has the focus now shifted from the continuous-time partition to discretized partitions? Was this paragraph about a discretized partition all along?

Reply:
Yes, that entire section was about "Clocked Discretized (Continuous-time) Partition" (see the following sub-sections). But for some reason "Clocked Discretized" was omitted in many places; and it was only described as continuous-time (since it is modelled as if continuous-time).

E.g., look at the following non-normative comment "It is not defined how to handle events that are triggered while solving the _continuous-time partition_. For example, a tool could handle events in the same way as for a usual simulation – but only check them at the time associated with clock-ticks."

That makes no sense for the normal continuous-time partition; where events should be handled normally and if taken literally it is not clear what clock-ticks they should be checked at.

The issue with "From the view of the continuous-time partition, the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must exactly hit." is similar, but subtler.

Reply:
@henrikt-ma is this clearer now?

Reply:
Yes. Resolving.